bin/cellranger_mtx_to_h5ad.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import scanpy as sc
+import argparse
+
+def mtx_to_adata( mtx_dir: str, sample: str, verbose: bool = False ):
+
+    if verbose:
+        print("Reading in {}".format(mtx_dir))
+
+    adata = sc.read_10x_mtx(mtx_dir)
```

Comment:
I am struggling in this one, because it seems that these functions treat the inputs and the object differently.
When I change the function as suggested, the conversion step works, but then the following error arises:

```python
Command error:
  Variable names are not unique. To make them unique, call `.var_names_make_unique`.
  Variable names are not unique. To make them unique, call `.var_names_make_unique`.
  Traceback (most recent call last):
    File "/Users/fal/Documents/GitHub/fal-scrnaseq/bin/concat_h5ad.py", line 38, in <module>
      adata = ad.concat(dict_of_h5ad, label="sample", merge="unique")
    File "/usr/local/lib/python3.8/site-packages/anndata/_core/merge.py", line 814, in concat
      alt_annot = merge_dataframes(
    File "/usr/local/lib/python3.8/site-packages/anndata/_core/merge.py", line 526, in merge_dataframes
      dfs = [df.reindex(index=new_index) for df in dfs]
    File "/usr/local/lib/python3.8/site-packages/anndata/_core/merge.py", line 526, in <listcomp>
      dfs = [df.reindex(index=new_index) for df in dfs]
    File "/usr/local/lib/python3.8/site-packages/pandas/util/_decorators.py", line 312, in wrapper
      return func(*args, **kwargs)
    File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 4176, in reindex
      return super().reindex(**kwargs)
    File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4811, in reindex
      return self._reindex_axes(
    File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 4022, in _reindex_axes
      frame = frame._reindex_index(
    File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 4041, in _reindex_index
      return self._reindex_with_indexers(
    File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4877, in _reindex_with_indexers
      new_data = new_data.reindex_indexer(
    File "/usr/local/lib/python3.8/site-packages/pandas/core/internals/managers.py", line 1301, in reindex_indexer
      self.axes[axis]._can_reindex(indexer)
    File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3476, in _can_reindex
      raise ValueError("cannot reindex from a duplicate axis")
  ValueError: cannot reindex from a duplicate axis
```

These are the files that have been produced using `sc.read_10x_h5` and used on the script if you wanna check:
[files.zip](https://github.com/nf-core/scrnaseq/files/8966693/files.zip)